src/lib/logging.ts

```diff
@@ -1,4 +1,79 @@
 import pino from 'pino';
+import {Dict} from '../types';
+
+export interface NodekitLogger {
+    info(message: string): void;
+    info(extra: Dict | undefined, message: string): void;
```

Comment:
Shouldn't message be first argument (for consistency with context class loggers)? Or it better be consistend with pino logger class methods?

Reply:
The intention was to be consistent with `pino` first.  I think it makes sense to do this rather than reorder the arguments of every call to the logger.